metadata-ingestion/src/datahub/ingestion/source/powerbi/config.py

```diff
@@ -105,6 +107,34 @@ class Constant:
     DATASET_WEB_URL = "datasetWebUrl"
 
 
+@dataclass
+class DataPlatformPair:
+    datahub_data_platform_name: str
+    powerbi_data_platform_name: str
+
+
+class SupportedDataPlatform(Enum):
```

Comment:
Nice! How about Redshift, Mysql?

Reply:
I merged the master redshift is now available. I need to open a separate PR for MySQL as currently connector doesn't support it